EnglishFilter/sections/specific.txt

```diff
@@ -155,6 +155,8 @@ nekopoi.care##div[class^="adsgen"]
 vladan.fr##a[href*="banner"][href*="utm_source"][target="_blank"]
 ||vladan.fr/wp-content/uploads/images/veeamskin3.png
 vladan.fr##a[href^="https://www.vladan.fr/go"] > img
+sinagritbabasworkshop.com##.widget-content > div[style="text-align:justify"] > a:not([href*="dmca.com"])
+sinagritbabasworkshop.com##.adsbygoogle
```

Comment:
##.adsbygoogle is already generic rule.

Reply:
Yesterday i had ad leftover by `.adsbygoogle`, that's why I add it. Today I can't reproduce it.